conf/db.sql

```diff
@@ -76,11 +76,11 @@ DROP TABLE IF EXISTS `Dialogs`;
 /*!40101 SET character_set_client = utf8 */;
 CREATE TABLE `Dialogs` (
   `DialogID` bigint(20) NOT NULL AUTO_INCREMENT,
-  `DialogNumber` bigint(20) NOT NULL DEFAULT '0',
+  `DialogNumber` bigint(20) NOT NULL DEFAULT 0,
   `DialogName` varchar(200) NOT NULL DEFAULT '',
   `LanguageCode` char(2) NOT NULL DEFAULT '',
-  `DialogText` text,
-  `DialogType` smallint(6) NOT NULL DEFAULT '0',
+  `DialogText` text DEFAULT NULL,
```

Comment:
This is actually not just a change due to a MariaDB upgrade. It seems to stand by itself, possibly some kind of bugfix.